app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -517,6 +519,15 @@ private String prettyCaption(Media media) {
         }
     }
 
+    private String prettyDepiction(Media media) {
```

Comment:
I don't think this method is needed. All depictions are pretty :-)

Reply:
@nicolas-raoul I did not understood this are you talking about the name of the method?

I added this method for this condition:
```
if (depiction.equals("")) {
            return getString(R.string.detail_depiction_empty);
```

Reply:
No depiction can be an empty string. If the API returns no label (should almost never happen), just use the QID such as "Q12280"

Reply:
okay